### DISCUSSION TITLE
不设防的交换 ———— 如何为自由/开源软件付‘费’ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#62] [Announcements]
# 不设防的交换 ———— 如何为自由/开源软件付‘费’ - 「开源之道」

https://opensourceway.community/posts/open-source-economic/how-to-pay-for-free-software/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

21年前就已经嗯哼过: [如何成为一名黑客 — Chinese Translations 0.1 documentation](https://translations.readthedocs.io/en/latest/hacker_howto.html)

付费中的费, 就是 fee 的音译 ;-)

而费用, 其实就是社会必要劳动时间,
这种时间可以是时间本身, 也可以是等价兑换物,
而 FLOSS 世界, 最欢迎以及受到尊敬的, 还是饱含智慧的劳动时间...

所以,

企业使用开源系统, 最应该的支付, 
其实就是认真使用, 合规反馈贡献;

其它无它...

**Reply by lijiangsheng1** [reply]
常识都需要我们花这么大的力气来普及～ 可见环境是多么的恶劣。

**Reply by ZoomQuiet** [reply]
#是也乎,(￣▽￣)
其实, 一个社会中常识之所以是常识,
是因为涉及的经验是太多数人都有经历过的,
而开源涉及的民主自治在...是很少有机会体验的...
所以,.,...这力气, 还真不能大太力...
